bin/martinize2

```diff
@@ -360,6 +360,10 @@ def entry():
                             default=[], action='append', type=lambda x: x.split(','),
                             help='Ignore residues with that name: e.g. '
                             '-ignore HOH,LIG (+)')
+    file_group.add_argument('-protres', dest='protres', default=False,
+                            action='store_true',
+                            help="Check if molecules have the different chains "
+                                 " or resids when merging and preserve them if they are")
```

Comment:
Help is missing an 'else'. I like the functionality, but I don't like the name of the flag. Can we think of something better?